README.md

```diff
@@ -155,6 +155,20 @@ The configuration file may also optionally contain the following fields:
 
 See [`testing-and-tuning/sample_conf.json`](https://github.com/mitre/data-owner-tools/blob/csv-translation/testing-and-tuning/sample_conf.json) for an example of the configuration file.
 
+### Data Quality and Characterization
+
+A data characterization script is provided to assist in identifying data anomalies or quality issues. This script can be run against the `pii.csv` generated by `extract.py` or directly against the database used by `extract.py`.
+It is recommended that `data_analyis.py` at least be run against the generated `pii.csv` file to help ensure that extraction succeeded successfully.
```

Comment:
Now that we have merged in #27, we should create a convention for referring to the `pii.csv` file since it will now be `pii-TIMESTAMP.csv` (maybe that's the convention).

Reply:
Agreed. There's no `-` in the pii.csv name currently (e.g. `pii20220902T090139.csv`), so `piiTIMESTAMP.csv` would be more accurate, but way uglier.

https://github.com/mitre/data-owner-tools/blob/8a313915344aab4e930c7f618330ef9eec4856bd/extract.py#L272

I move that we add a `-` for increased readability and adopt the `pii-TIMESTAMP.csv` convention as you suggested.

Reply:
+1

Reply:
Done in https://github.com/mitre/data-owner-tools/pull/29/commits/308bf57b0064bdbde46cbad58089ea58cf47d77b